domains/primitives/auto-id/src/runtime_interface.rs

```diff
@@ -1,26 +1,46 @@
-#[cfg(feature = "std")]
-use crate::host_functions::HostFunctionExtension;
 use crate::{DerVec, SignatureVerificationRequest, TbsCertificate};
-#[cfg(feature = "std")]
-use sp_externalities::ExternalitiesExt;
 use sp_runtime_interface::runtime_interface;
 
 /// AutoId runtime interface.
 #[runtime_interface]
 pub trait AutoIdRuntimeInterface {
     fn verify_signature(&mut self, req: SignatureVerificationRequest) -> Option<()> {
-        self.extension::<HostFunctionExtension>()
-            .expect(
-                "No `CertificateRegistryHostFunctionExtension` associated for the current context!",
-            )
-            .verify_signature(req)
+        // TODO: we need to conditional compile for benchmarks here since
+        //  benchmark externalities does not provide custom extensions.
+        //  Remove this once the issue is resolved: https://github.com/paritytech/polkadot-sdk/issues/137
```

Comment:
This and below, the white space was intentional to ensure the the entire comment belongs to TODO rather than seperate comment y itself.

My IDE shows this as a full TODO description with white space. With your changes, it would not